### DISCUSSION TITLE
DeepONet

### DISCUSSION THREAD

**Discussion by jcddafonseca** [lululxvi/deepxde#1545] [General]
Hi!

 I'm trying to use the DeepONet class to learn an operator G: u -->v, where "v" is the solution of a 2D Helmholtz equation and "u" is a given "source" function in the PDE. So, the input function "u" is defined in a 2D domain. I don't know any solution (or demo) using DeepONet from DeepXDE  for a problem where the domain of the input function has more than one dimension.
Could anybody tell me if there is an example and where to find it? I also wonder how the GRF2D class could be properly used to deal with this kind of problem.

Thanks,
Júlio

**Comment by lululxvi** [comment]
- https://github.com/lululxvi/deeponet
- https://github.com/lu-group/
